external/frozen/include/frozen/bits/pmh.h

```diff
@@ -125,7 +131,7 @@ struct seed_or_index {
   using value_type = uint64_t;
 
 private:
-  static constexpr value_type MINUS_ONE = -1;
+  static constexpr value_type MINUS_ONE = std::numeric_limits<value_type>::max();
```

Comment:
Why -1 become std::numeric_limits<value_type>::max(); ?

Reply:
It's their code. I think value_type is size_t, and they want avoid warning